bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -57,6 +57,7 @@ public class FormContainerImpl extends AbstractContainerImpl implements
 
     private static final String DOR_TYPE = "dorType";
     private static final String DOR_TEMPLATE_REF = "dorTemplateRef";
+    private static final String DOR_TEMPLATE_CHANGED = "dorTemplateChanged";
```

Comment:
where is this used?

Reply:
This property is used to show message if acroForm templates is changed and review the bindings
Currenlty this properties is saved in asset node of form, but going forward with sites  we want this to persist in form conatiner node. So exposing API to read this property.